plugins/score-card/src/api/ScoringDataJsonClient.ts

```diff
@@ -72,10 +72,10 @@ export class ScoringDataJsonClient implements ScoringDataApi {
     return this.extendEntityScore(result, undefined);
   }
 
-  public async getAllScores(): Promise<EntityScoreExtended[] | undefined> {
+  public async getAllScores(entityKinds?: ScorableEntityKind[]): Promise<EntityScoreExtended[] | undefined> {
```

Comment:
just a thought... how about `entityKindFilter`? Have you considered this as well?

Reply:
Haven't consider it.
It's is more explicit :+1:

Reply:
Thanks, closing!